### ISSUE TITLE
Implementations of this?

### ISSUE THREAD

**Issue by bradib0y** [geohot/fromthetransistor#9]
I'm looking forward to see someone who knows better than me giving advice: are there any working implementations of this curriculum assembly?

I'd be happy to participate. I'm into IT/CS/Programming since 2012 approximately, I think something like this would really cement my knowledge and understanding. If everything were tailored for beginners, I would still go through it, just in case if I'm missing something.

I'm always looking for the next big thing to learn, but there's just so many. I'm thinking of chasing a PHd in applied maths or electrical engineering, both for the sake of actually going deeper with computer science as an interdisciplinary field. But something like this could be a real gamechanger since it is more focused towards my interests.

**Comment by kfur** [comment]
Begin from Digital Design and Computer Architecture book.

**Comment by elimisteve** [comment]
Yes, see Nand2Tetris for a similar course: https://github.com/geohot/fromthetransistor/issues/13

**Comment by A-janjan** [comment]
I recommend you to see [this repository](https://github.com/tuhdo/os01); it may be helpful for you.
